pakwheels_clone/cars/models.py

```diff
@@ -0,0 +1,57 @@
+from django.contrib import admin
+from django.db import models
+
+
+from users.models import User
+
+
+class CarManufacturer(models.Model):
+    car_make = models.CharField(max_length=50)
+
+
+class CarModel(models.Model):
+    car_model = models.CharField(max_length=150)
+
+
+class CarFeatures(models.Model):
+    feature = models.CharField(max_length=200)
+
+
+class CarImages(models.Model):
+    image = models.ImageField(upload_to="images")
+
+
+class CarDetails(models.Model):
+    fuel_choices = [
```

Comment:
django provides more intuitive and new ways for model choices see [here](https://docs.djangoproject.com/en/5.0/ref/models/fields/#enumeration-types)